### DISCUSSION TITLE
Make Accuracy Challenge mod rankable

### DISCUSSION THREAD

**Discussion by SevenChords** [ppy/osu#26919] [Ideas] [answered]
rn any play with the accuracy challenge mod enabled doesn't give pp, which imho is unnecessary since the accuracy challenge mod doesn't affect gameplay/ranking/score in any way except auto restarting when you cannot reach your desired accuracy anymore, which can be done manually if one wanted to, so i think it would be fair to make it a rankable mod with a score modifier of 1.00x and enable pp for plays with the mod enabled.

**Comment by bdach** [comment, marked as answer]
https://github.com/ppy/osu/pull/26935
